toolkit/activity.py

```diff
@@ -146,18 +150,18 @@ def get_cursor(self):
         return self._cursor
 
     def set_cursor(self, cursor):
-        if not isinstance(cursor, gtk.gdk.Cursor):
+        if not isinstance(cursor, Gdk.Cursor):
             cursor = CursorFactory().get_cursor(cursor)
 
         if self._cursor != cursor:
             self._cursor = cursor
-            self.window.set_cursor(self._cursor)
+            self.get_window().set_cursor(self._cursor)
```

Comment:
GTK 3 does not make cursor changes immediately.  See Activity.busy for how to flush this.

Reply:
Activity.busy where it can be found?

Reply:
See [API](https://developer.sugarlabs.org/sugar3/sugar3.activity.activity.html) and search for busy.

See [Activity.busy](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/activity/activity.py#L1402).

It is the [call to _Gdk.Flush_](https://github.com/sugarlabs/sugar-toolkit-gtk3/blob/master/src/sugar3/activity/activity.py#L1435) needed in recent GTK releases, unless the application uses a spinner or progress bar.

Reply:
Tried it in 318b019.